<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRoundDialog</class>
 <widget class="QDialog" name="NewRoundDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Round</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel { color: white; }
QDialog { background: rgb(51,51,51)}
QCheckBox { color: white; }</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonNormal">
       <property name="text">
        <string>Normal Round</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonHochzeit">
       <property name="text">
        <string>Hochzeit</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSolo">
       <property name="text">
        <string>Solo</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonTrumpfabgabe">
       <property name="text">
        <string>Trumpfabgabe</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Re:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PlayersComboBox" name="comboBoxNormalRe1"/>
       </item>
       <item row="1" column="1">
        <widget class="PlayersComboBox" name="comboBoxNormalRe2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Schweine:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="PlayersComboBox" name="comboBoxNormalSchweine"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Points:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="spinBoxNormalPoints">
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Winner:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="comboBoxWinner"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QTextEdit" name="textEditNormalComment"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>GamePoints:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QSpinBox" name="spinBoxNormalRePoints">
           <property name="maximum">
            <number>240</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxNormalContraPoints">
           <property name="maximum">
            <number>240</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Hochzeit:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PlayersComboBox" name="comboBoxHochzeitHochzeit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Schweine:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="PlayersComboBox" name="comboBoxHochzeitSchweine"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Points:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="spinBoxHochzeitPoints">
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Winner:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="comboBoxHochzeitWinner"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QTextEdit" name="textEditHochzeitComment"/>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="PlayersComboBox" name="comboBoxHochzeitFellow"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxHochzeitDecision">
           <item>
            <property name="text">
             <string>Erster Fehl</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Erster Trumpf</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Fellow Player:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>GamePoints:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QSpinBox" name="spinBoxHochzeitRePoints">
           <property name="maximum">
            <number>240</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxHochzeitContraPoints">
           <property name="maximum">
            <number>240</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Solo:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PlayersComboBox" name="comboBoxSoloPlayer"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Solo type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxSoloType"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Schweine:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="PlayersComboBox" name="comboBoxSoloSchweine"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Points:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="spinBoxSoloPoints">
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QTextEdit" name="textEditSoloComment"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Pflicht:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkBoxSoloPflicht">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="comboBoxSoloWinner"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Winner:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="PlayersComboBox" name="comboBoxSoloHochzeit"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxSoloHochzeitDecision">
           <item>
            <property name="text">
             <string>---</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Erster Fehl</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Erster Trumpf</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Hochzeit:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>GamePoints:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QSpinBox" name="spinBoxSoloRePoints">
           <property name="maximum">
            <number>240</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxSoloContraPoints">
           <property name="maximum">
            <number>240</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Trumpfabgabe:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Accepted by:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PlayersComboBox" name="comboBoxTrumpfabgabePlayer"/>
       </item>
       <item row="1" column="1">
        <widget class="PlayersComboBox" name="comboBoxTrumpfabgabeAccept"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Schweine:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="PlayersComboBox" name="comboBoxTrumpfabgabeSchweine"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Points:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QTextEdit" name="textEditTrumpfabgabeComment">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.Lucida Grande UI'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Lucida Grande';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="spinBoxTrumpfabgabePoints">
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Winner:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="comboBoxTrumpfabgabeWinner"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Trumpf Count:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="spinBoxTrumpfCount">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>3</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxTrumpfZurueck">
           <property name="text">
            <string>Trumpf zurück</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>GamePoints:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QSpinBox" name="spinBoxTrumpfabgabeRePoints">
           <property name="maximum">
            <number>240</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxTrumpfabgabeContraPoints">
           <property name="maximum">
            <number>240</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlayersComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">ui/widgets/playerscombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
